examples/error-user-defined/error_user_defined.bal

```diff
@@ -0,0 +1,107 @@
+import ballerina/io;
```

Comment:
Shall we rename this directory/files as user-defined-errors/user_defined_errors? 

(Given that there is no grouping by name already)